requirements.txt

```diff
@@ -7,6 +7,7 @@ jsonschema==4.19.2
 numpy==1.26.2
 openai>=1.3.5
 anthropic>=0.20.0
+google-generativeai
```

Comment:
Can you please pin the version

Reply:
[ddd3e86](https://github.com/thesofakillers/aideml/pull/5/commits/ddd3e86546fce7c974d4e14c8b90f6a7fd97773a)